src/components/team-members/team-members.js

```diff
@@ -1,25 +1,25 @@
-import React from 'react';
-import TeamMember from '../team-member/team-member';
-import { TEAM_MEMBERS } from '../../data/team-members';
-import './team-members.scss';
+import React from "react";
+import TeamMember from "../team-member/team-member";
+import { TEAM_MEMBERS } from "../../data/team-members";
+import "./team-members.scss";
 
```

Comment:
can we keep the quotes to be single quotes here and everywhere

Reply:
Yeah sure, sorry my prettier might have corrected them. Will change 👍